shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/data/network/builder/HttpRequestBuilder.kt

```diff
@@ -0,0 +1,8 @@
+package co.nimblehq.ic.kmm.suv.data.network.builder
+
+import co.nimblehq.ic.kmm.suv.BuildKonfig
+import io.ktor.client.request.*
+
+fun HttpRequestBuilder.path(path: String) {
+    url(BuildKonfig.BASE_URL.plus(path))
+}
```

Comment:
No usage yet, will we use it somewhere or I'm missing its usage? 🤔

Reply:
Please ignore this. It will be used in the next PR when we call API, this is more helpers function, so I decided to put it here (Update the description of this PR to contains dependencies and helpers)